src/sentry/static/sentry/app/components/gridEditable/styles.tsx

```diff
@@ -116,7 +117,17 @@ export const GridHeadCellButton = styled('div')<GridEditableProps>`
   text-overflow: ellipsis;
   overflow: hidden;
 
-  background: ${p => (p.isEditing ? p.theme.offWhite2 : 'none')};
+  background: ${p => {
+    if (p.isDragging) {
+      return '#CEC2D8';
```

Comment:
Should this be a theme value or use `color` to derive from a theme value?

Reply:
Updated in https://github.com/getsentry/sentry/pull/14980/commits/c6a43812578e55fe517681ff2feeeb1a677ec3ae